packages/read-and-create-calendar-events/client/react/src/App.js

```diff
@@ -140,33 +148,45 @@ function Agenda({ serverBaseUrl, userId, calendarId }) {
 
   useEffect(() => {
     const getCalendarEvents = async () => {
-      try {
-        const url = `${serverBaseUrl}/nylas/read-events?${
-          calendarId ? 'calendarId=' + calendarId : ''
-        }`;
-
-        const res = await fetch(url, {
-          method: 'GET',
-          headers: {
-            Authorization: userId,
-            'Content-Type': 'application/json',
-          },
-          params: {
-            calendarId,
-          },
-        });
-
-        if (!res.ok) {
-          throw new Error(res.statusText);
+      if (calendarId) {
+        try {
+          const startsAfter = null; // applyTimezone(getLocalDateString(new Date(new Date().setDate(new Date().getDate() - 7))));
+          const endsBefore = applyTimezone(
+            getLocalDateString(
+              new Date(new Date().setDate(new Date().getDate() + 7))
+            )
+          );
```

Comment:
Please simplify this, make a util function and be state purpose with the function name.